rivr/wsgi.py

```diff
@@ -151,16 +151,27 @@ def body(self):
 
         return self.environ['wsgi.input']
 
-    #@property
-    def get_get(self):
-        if not hasattr(self, '_get'):
-            self._get = dict((k, v) for k, v in parse_qsl(self.environ.get('QUERY_STRING', ''), True))
-        return self._get
-    GET = property(get_get)
+    @property
+    def parameters(self):
```

Comment:
My main concern is that parameters is not the right word for this, because parameters can be more than just the query parameters, but also other URI parameters (`/questions/{id}`).